waffle-chai/src/inject-call-history.ts

```diff
@@ -29,10 +29,18 @@ function toRecordedCall(message: any): RecordedCall {
 
 const inject = () => {
   let waffle: any;
+  let config: any;
   try {
-    waffle = require('hardhat')?.waffle;
+    ({waffle, config} = require('hardhat'));
   } catch { return; }
   if (!waffle || !waffle.provider) return;
+  if (!config) {
+    console.warn('Missing hardhat config, skipping call history injection');
```

Comment:
```suggestion
    console.warn('Hardhat config not present, skipping call history injection');
```

Use lighter wording - "missing" suggests it's a mistake and the user should add it - but if the user uses Ganache then it is not a mistake.